### DISCUSSION TITLE
file ownership / permission denied on created videos

### DISCUSSION THREAD

**Discussion by prismspecs** [AbdBarho/stable-diffusion-webui-docker#557] [General]
If I try to delete a file from nautilus, or to open the video with VLC I get permission errors, since it is not owned by my main computer user but I suppose the docker container specified user. Do you have any advice for this?

**Comment by AbdBarho** [comment]
`sudo rm ...`

**Reply by prismspecs** [reply]
not sure if you are trolling me trying to get me to delete my files, but the issue is not that I want to remove files but that the docker container takes ownership of the files created therein.

**Reply by AbdBarho** [reply]
the container created these files so it has ownership.
the container runs as root so the files are owned by the root user.

there is a PR to use non-root user but there are still open questions: https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/546

If you want to change ownership (assuming WSL):
```
sudo chown -R $(id -u):$(id -g) [your folder here]
```

**Comment by DevilaN** [comment]
What kind of help or solution do you expect?
